pwa/src/templates/templateParts/formHeader/FormHeaderTemplate.module.css

```diff
@@ -30,3 +31,13 @@
 .buttonsContainer > *:not(:last-child) {
   margin-inline-end: var(--gateway-ui-size-xs);
 }
+@media only screen and (min-width: 992px) {
+  .container {
+    flex-direction: row;
+  }
+
+  .content {
+    overflow-y: auto;
+    width: 100%;
+  }
```

Comment:
There's no element with the `.content` className in this component (I think). So I think these rules are redundant.